packages/suite/src/middlewares/suite/suiteMiddleware.ts

```diff
@@ -71,6 +71,7 @@ const suite =
         }
 
         if (deviceActions.connectDevice.match(action)) {
+            // TODO: should this apply also for DeviceModelInternal.T3B1 ???
```

Comment:
I would expect T3T1 to have similar feature set that T3B1 but I am not sure.

Reply:
You mean similar to T2B1, right?